profile/README.md

```diff
@@ -8,4 +8,10 @@ The Axon Ivy Market is from developers for developers 👩‍💻. Any kind of c
 is very welcome. Developing a small new feature 🏎️, filing an issue 📝, fxing a bug 🪲 or even
 [creating a completely new connector](https://github.com/axonivy-market/market/wiki) 🚀.
 
-Be part of the Axon Ivy Community and get involved!
+**Be part of the Axon Ivy Community and get involved!**
+
+Tools:
```

Comment:
having a section to found all the utilities around the market would benefit new users from our team to find the stuff.
so to have it prominent I thought about the github.profile. ... 
I think it will also not harm normal users that look around here @alexsuter , but if you feel like this developers infos must live somewhere else: just share your ideas.